### ISSUE TITLE
torch.nn.functional.normalize get nan

### ISSUE THREAD

**Issue by so-coolboy** [pytorch/pytorch#142302]
### 🐛 Describe the bug

when i run this code：
<img width="728" alt="image" src="https://github.com/user-attachments/assets/2e839b7d-62f3-47ca-95f0-7c710e39ed79">

   i found p_reps[4:,] is nan as :
<img width="517" alt="image" src="https://github.com/user-attachments/assets/6efb9dbc-65d4-43af-9165-949596b8850c">
even if i add eps=1e-2, it was unuseful.
<img width="502" alt="image" src="https://github.com/user-attachments/assets/de3287af-66cb-4043-a1ff-59fff501e600">
how can i do to solve ?

### Versions

1

**Comment by mikaylagawarecki** [comment]
Could you provide a minimal snippet to repro this please
